MODULE.bazel

```diff
@@ -0,0 +1,20 @@
+module(name = "jsonnet_go", version = "0.0.0")
+
+bazel_dep(name = "gazelle", version = "0.30.0", repo_name = "bazel_gazelle")
+bazel_dep(name = "jsonnet", version = "0.0.0", repo_name = "cpp_jsonnet")
+bazel_dep(name = "rules_go", version = "0.39.1", repo_name = "io_bazel_rules_go")
+
+git_override(
+    module_name = "jsonnet",
+    remote = "https://github.com/mortenmj/jsonnet.git",
+    commit = "df811822d9a31cb3fab9bdf68679ad300f1c12ed",
+)
```

Comment:
This would be updated once https://github.com/google/jsonnet/pull/1083 lands. This PR is not ready until that happens.